packages/frontend/src/components/UserSettings/AccessTokensPanel/TokensTable.tsx

```diff
@@ -0,0 +1,124 @@
+import { Button, Classes, Dialog } from '@blueprintjs/core';
+import { ApiPersonalAccessTokenResponse, formatDate } from '@lightdash/common';
+import {
+    Button as MantineButton,
+    createStyles,
+    Paper,
+    Table,
+} from '@mantine/core';
+import { FC, useState } from 'react';
+import {
+    useAccessToken,
+    useDeleteAccessToken,
+} from '../../../hooks/useAccessToken';
+
+const useTableStyles = createStyles((theme) => ({
+    root: {
+        '& thead tr': {
+            backgroundColor: theme.colors.gray[0],
+        },
+
+        '& thead tr th': {
+            color: theme.colors.gray[6],
+            fontWeight: 600,
+        },
+
+        '& thead tr th, & tbody tr td': {
+            padding: '12px 20px',
+        },
+
+        '& tr td:last-child': {
+            textAlign: 'right',
+        },
+
+        '&[data-hover] tbody tr': theme.fn.hover({
+            cursor: 'pointer',
+            backgroundColor: theme.fn.rgba(theme.colors.gray[0], 0.5),
+        }),
```

Comment:
this is the same done in `packages/frontend/src/components/common/ResourceView/ResourceViewList/index.tsx`

with the exception of font-size not changing, and a text align right to the last td of a tr.